pywal/__main__.py

```diff
@@ -15,11 +15,22 @@
 import shutil
 import sys
 
-try:
-    import colorama
-    colorama.just_fix_windows_console()
-except ImportError:
-    colorama = None
+if sys.platform.startswith("win"):
+    try:
+        import colorama
+        colorama.just_fix_windows_console()
+    except ImportError:
+        no_colorama_terms = [
+                "wezterm",
+                "alacritty",
+                "hyper",
+                "putty",
+                "ghostty",
+                "mintty",
+                ]
+        winterm = os.environ.get("TERMINAL")
+        if winterm not in no_colorama_terms:
+            show_colorama_warning = True
```

Comment:
I think this should be set to False or None much earlier, in case this block is not hit...

Reply:
https://github.com/eylles/pywal16/pull/152/commits/67af8f0662e1528f943dd4e2bd295a41b03c5302 should address it